collection-scripts/gather_vsphere_config

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+VSPHERE_CONFIG_EVENTS_PATH=$BASE_COLLECTION_PATH/vsphere_config_events
+
+if [ -z "$(oc get deployment -n openshift-cluster-storage-operator vsphere-problem-detector-operator | grep vsphere-problem-detector-operator)" ]; then
+exit
+fi
+
+METRICS="vsphere_vcenter_info vsphere_features vsphere_cluster_check_total vsphere_cluster_check_errors vsphere_node_check_errors vsphere_node_check_total vsphere_node_hw_version_total"
```

Comment:
What of these should we be collecting with insights/telemetry vs the must-gater archive?

Reply:
> I want to have @radekvokal and @mfojtik review this as I feel like metrics collection may want to be in one script vs N (individual scripts).

That sounds reasonable to me.  Definitely don't want to go down a custom path here if it's not necessary.

Reply:
Not sure if 'custom path' is the wording I would choose here, but there may be a different approach to take when it comes to 'metrics' collection. Its why I want our folks in CCX to take a look t this, as well as for @mfojtik (as we may want to have a 'generic' collect_metrics script (that simply lists all the metrics we want data on (think the `gather` script, a resource builder that then collects all of what we want in an efficient way).